### PR TITLE
Add Extension 

### PR DESCRIPTION
**Pull request by MeLoseAgain**
Closes: #15654 
Closes: #15554 
Closes: #15433
Closes: #15337 
Closes: #15329 
Closes: #15299 
Closes: #15132 
Closes: #14865

I fixed the icon for inazumanga which uses the default icon, added res to the folder

Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [ ] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [ ] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

File: multisrc/overrides/madara/freemangatop/res/mipmap-hdpi/ic_launcher.png
Comment (alessandrojean) [inline multisrc/overrides/madara/freemangatop/res/mipmap-hdpi/ic_launcher.png:1]: You don't need to add the icon from the base theme to the overrides, can remove these.

**Review by alessandrojean — CHANGES_REQUESTED** [review]

**Comment by alessandrojean** [comment]
About https://github.com/tachiyomiorg/tachiyomi-extensions/issues/15433, you need to remove/move the existing extension as well. Move from mangafront to Madara.

File: multisrc/overrides/madara/freemangatop/res/mipmap-hdpi/ic_launcher.png
Comment (MeLoseAgain) [reply multisrc/overrides/madara/freemangatop/res/mipmap-hdpi/ic_launcher.png:1]: > You don't need to add the icon from the base theme to the overrides, can remove these.

I'm moving from inazumanga/ to inazumanga/res so is that a problem

Comment (alessandrojean) [reply multisrc/overrides/madara/freemangatop/res/mipmap-hdpi/ic_launcher.png:1]: You don't need these files as they're the same default ones from the theme. They will be copied during the build.

If you added it to a wrong folder in a previous commit, just delete them in a new commit.

**Review by alessandrojean — APPROVED** [review]

**Comment by MeLoseAgain** [comment]
Wait a minute

**Comment by stevenyomi** [comment]
Better split into 3 PRs (one for each multisrc).

**Comment by MeLoseAgain** [comment]
Is that so? I don't mind deleting it (⁠≧⁠▽⁠≦⁠)

**Pull request merged**